.ci/gcb-build-container.yml

```diff
@@ -0,0 +1,15 @@
+---
+# substitutions:
+#   _IMAGE_NAME: 'gcr.io/graphite-docker-images/bash-plus:latest'
+#   _DIRECTORY: '.ci/containers/bash-plus'
+
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-t', '$_IMAGE_NAME', '.']
+    dir: '$_DIRECTORY'
+
+images:
+  - '$_IMAGE_NAME'
```

Comment:
where are we getting this value?

Reply:
okay, found the `bash-plus-test` trigger. Oh, I see, so we're gonna have separate triggers for each container but share the building yaml file?

Reply:
Yep !.. Substitutions are in the trigger UI